### ISSUE TITLE
Presort manifests (preferably by module name)

### ISSUE THREAD

**Issue by rschristian** [es-tooling/module-replacements#119]
It'd be nice if the manifests were presorted (preferably by module name) so that consumers could do a binary search over the list(s). It's also a bit nicer to manually search (w/out ctrl+f) if it's alphabetical.

Of course, this isn't not without it's own maintenance burdens so I get this is not a guarantee anyone wants to make.

**Comment by 43081j** [comment]
makes sense but i think we should do it at build or commit time

e.g. we could have a CI step to check they're sorted, and an npm script to sort them

**Comment by rschristian** [comment]
Yup, that's what I was thinking. I'd personally lean towards commit time, to give that extra readability to the source manifests too, but happy either way if you're willing!

I can take a look at adding something tomorrow.

**Comment by 43081j** [comment]
a CI check would be nice still to fail builds which didn't sort

so i think maybe:

- add a script to sort the manifests and write back to the files
- add a CI step to run this in "check" mode, i.e. lint that they are sorted (and fail CI if they're not)

we can then use the same script for linting other things. for example, it'd be nice to check _new entries_ actually exist in npm at ci-time

**Comment by rschristian** [comment]
Sorry, a CI script would be included in that, yeah. I was distinguishing pre-publish from source, not the method of enforcing the sorting.
